plaidml/edsl/__init__.py

```diff
@@ -881,12 +881,19 @@ class NearestMode(enum.Enum):
     SIMPLE = 4
 
 
+class GatherMode(enum.Enum):
+    NORMAL = 0
+    ND = 1
+
+
 def gather(x,
            y,
            axis=0,
            interpolation_mode=InterpolationMode.LINEAR,
            nearest_mode=NearestMode.ROUND_PREFER_FLOOR,
-           cube_coeff=-0.75):
+           cube_coeff=-0.75,
+	   gather_mode=GatherMode.NORMAL,
```

Comment:
Please format with the same indent.

Reply:
I'm not sure how to format python file. Seems clang-format doesn't apply.

Reply:
You can use yapf -i yourfile

Reply:
Actually when you commit, pre-commit hooks should run yapf. You can run pre-commit to double check.